Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -757,6 +757,35 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("object", _("Object"), "Drawer")
       .AddParameter("expression", _("X drawing position"))
       .AddParameter("expression", _("Y drawing position"));
-
+  obj.AddAction("SetAntialiasingQuality",
+                _("Anti-aliasing"),
+                _("Anti-aliasing"),
+                _("Set anti-aliasing of _PARAM0_ to _PARAM1_"),
+                _("Setup"),
+                "res/actions/antialiasing24.png",
+                "res/actions/antialiasing.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .AddParameter("stringWithSelector",
+                    _("Anti-aliasing type"),
+                    "[\"Off\",\"Low\",\"Medium\",\"High\"]");
+  obj.AddCondition(
+         "CheckAntialiasingQuality",
+         _("Anti-aliasing type"),
+         _("Checks the selected type of anti-aliasing"),
+         _("The anti-aliasing of _PARAM0_ is set to _PARAM1_"),
+         _("Setup"),
+         "res/actions/antialiasing24.png",
+         "res/actions/antialiasing.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .AddParameter("stringWithSelector",
+                    _("Type of anti-aliasing to check the object against"),
+                    "[\"Off\",\"Low\",\"Medium\",\"High\"]");
+  obj.AddStrExpression(
+         "GetAntialiasingQuality",
+         _("Type of anti-aliasing used by a shape painter"),
+         _("Gets the type of anti-aliasing in use: Off, Low, Medium, or High."),
```

Comment:
```suggestion
         _("Return the type of anti-aliasing in use: Off, Low, Medium, or High."),
```

Reply:
The extension best practices say never to use "Returns ..." As an expression description though